MultiVersions/Forge/main/beachparty/lang/zh_tw.json

```
@@ -1,125 +1,161 @@
-  "block.beachparty.palm_beam": "棕櫚木梁柱",
+  "block.beachparty.palm_bar": "棕櫚木吧檯",
+  "block.beachparty.palm_bar_stool": "棕櫚木吧檯高腳凳",
   "block.beachparty.palm_button": "棕櫚木按鈕",
+  "block.beachparty.palm_cabinet": "棕櫚木櫥櫃",
+  "block.beachparty.palm_chair": " 棕櫚木椅",
   "block.beachparty.palm_door": "棕櫚木門",
   "block.beachparty.palm_fence": "棕櫚木柵欄",
   "block.beachparty.palm_fence_gate": "棕櫚木柵欄門",
   "block.beachparty.palm_floorboard": "棕櫚木地板",
+  "block.beachparty.palm_glass": "棕櫚木玻璃",
+  "block.beachparty.palm_glass_pane": "棕櫚木玻璃片",
   "block.beachparty.palm_hanging_sign": "懸掛式棕櫚木告示牌",
+  "block.beachparty.palm_hanging_wall_sign": "懸掛式棕櫚木告示牌",
   "block.beachparty.palm_leaves": "棕櫚木樹葉",
   "block.beachparty.palm_log": "棕櫚木原木",
   "block.beachparty.palm_planks": "棕櫚木材",
   "block.beachparty.palm_pressure_plate": "棕櫚木壓力板",
-  "block.beachparty.palm_sapling": "棕櫚木樹苗",
   "block.beachparty.palm_sign": "棕櫚木告示牌",
   "block.beachparty.palm_slab": "棕櫚木半磚",
+  "block.beachparty.palm_sprout": "棕櫚木樹苗",
   "block.beachparty.palm_stairs": "棕櫚木階梯",
-  "block.beachparty.palm_tall_torch": "棕櫚木高火把",
+  "block.beachparty.palm_table": "棕櫚木桌",
   "block.beachparty.palm_torch": "棕櫚木火把",
-  "block.beachparty.palm_torch_item": "棕櫚木火把",
   "block.beachparty.palm_trapdoor": "棕櫚木地板門",
-  "block.beachparty.palm_wall_torch": "牆上的棕櫚木火把",
+  "block.beachparty.palm_wall_sign": "牆上的棕櫚木告示牌",
   "block.beachparty.palm_wood": "棕櫚木塊",
   "block.beachparty.pumpkin_cocktail": "南瓜雞尾酒",
   "block.beachparty.radio": "收音機",
-  "block.beachparty.sand_bucket": "沙桶",
-  "block.beachparty.sand_bucket_block": "沙桶",
+  "block.beachparty.sand_bucket_block_empty": "空沙桶",
+  "block.beachparty.sand_bucket_block_filled": "裝滿的沙桶",
+  "block.beachparty.sand_bucket_empty": "空沙桶",
+  "block.beachparty.sand_bucket_filled": "裝滿的沙桶",
   "block.beachparty.sand_pile": "沙堆",
   "block.beachparty.sandcastle": "沙堆城堡",
-  "block.beachparty.sandwaves": "海浪沙",
+  "block.beachparty.sandwaves": "沙波浪",
+  "block.beachparty.seashell_block": "貝殼",
   "block.beachparty.stripped_palm_log": "剝皮棕櫚木原木",
   "block.beachparty.stripped_palm_wood": "剝皮棕櫚木塊",
   "block.beachparty.sweetberries_cocktail": "甜莓雞尾酒",
-  "block.beachparty.table": "棕櫚木桌",
-  "block.beachparty.tiki_bar": "熱帶風格吧檯",
-  "block.beachparty.tiki_chair": "熱帶風格吧檯椅",
-  "config.beachparty.subCategory.beach": "沙灘派對套裝",
-  "creativetab.beachparty.tab": "共度時光｜沙灘派對",
-  "entity.minecraft.villager.beachparty.barkeeper": "酒保",
-  "entity.minecraft.villager.beachparty.beach_boy": "沙灘男孩",
-  "entity.minecraft.villager.beachparty.beach_guy": "沙灘男孩",
-  "gui.beachparty.recipebook.toggleRecipes.refridgable": "可冷藏",
-  "gui.beachparty.recipebook.toggleRecipes.shakeable": "可調製",
+  "block.beachparty.tall_palm_torch": "高棕櫚木火把",
+  "block.beachparty.thatch": "茅草堆",
+  "block.beachparty.thatch_slab": "茅草堆半磚",
+  "block.beachparty.thatch_stairs": "茅草堆階梯",
+  "block.beachparty.wet_hay_block": "濕乾草方塊",
+  "creativetab.beachparty.tab": "[Let's Do] Beachparty",
```

一點吹毛求疵，~~開創造模式的時候，我希望我看到的模組名稱全是中文~~
如果接受的話，其餘的《Beachparty》看要不要翻成中文
```suggestion
  "creativetab.beachparty.tab": "共度時光｜沙灘派對",
```